docs/beta-launch/avasummit/overview.md

```diff
@@ -0,0 +1,127 @@
+
+- `uint16 requestConfirmations`: How many confirmations the Chainlink node should wait before responding. The longer the node waits, the more secure the random value is. It must be greater than the `minimumRequestBlockConfirmations` limit on the coordinator contract, which is `1` on Avalanche Fuji Testnet. In this example, we set the value to `1,` but you can set it higher.
+
+- `uint16 numWords`: How many random values to request. If you can use several random values in a single callback, you can reduce the amount of gas you spend per random value. The total cost of the callback request depends on how your `fulfillRandomWords()` function processes and stores the received random values, so adjust your `callbackGasLimit` accordingly. Here we are requesting two random values.
+
+To use this contract:
+
+1. Open the [contract in Remix](https://remix.ethereum.org/#url=https://docs.chain.link/samples/beta-launch/avasummit/VRFv2SubscriptionManager.sol).
+
+1. Compile and deploy the contract using the Injected Web3 environment. The contract includes all the configuration variables you need, but you can edit them if necessary (cf. above).
+
+This contract automatically creates a new subscription when you deploy it. Read the `s_subscriptionId` variable to find the contract subscription ID.
+
+1. Fund your contract with at least two testnet LINK as shown [here](/docs/fund-your-contract/). Call the `getLinkBalance()` function to verify: it should return `2000000000000000000`.
+
+1. In this example, the `topUpSubscription()` function sends LINK from your contract to the subscription. Fund your contract with at least two testnet LINK. If you need testnet LINK, you can get it from one of the available [Avalanche Fuji faucets](/docs/link-token-contracts/#fuji-testnet).
+   Run the `topUpSubscription()` function to send LINK from your contract to its subscription balance. For this example, specify a value of `2000000000000000000`, which is equivalent to two LINK.
+
+1. Call the `getSubscriptionDetails()` function to get details of the subscription. You will notice that your contract is part of the consumers list and that its subscription balance is 2 LINK.
+
+1. On the consumer contract, run the `requestRandomWords()` function to request and receive random values. The request might take several minutes to process. You can track if the request is still pending by calling `pendingRequestExists().` When `pendingRequestExists()` returns `false`, that means `fulfillRandomWords()` has been called. `fulfillRandomWords()` receives random values and stores them in the contract.
+
+1. Read `s_randomWords` at index `0`or `1`to fetch the random values.
```

Comment:
```suggestion
1. Because this contract requested two random values, the oracle returns an array with two values. Read `s_randomWords` at index `0` or `1`to fetch the random values.
```